### ISSUE TITLE
Improve build documentation and Error Handling

### ISSUE THREAD

**Issue by StoreBoughtRocketGames** [jordigcs/fmod_gd4#1]
You might want to add into the instructions a step to copy the fmodL.dll, and the fmodstudioL.dll into the build bin folder. I kept getting crash errors when trying to run Godot until I went into the fmod api and found those dll's and put them in the bin folder (which was just a guess on my part from some past c++ usage). Hopefully that's the right thing to do. I haven't gotten a chance to test out the API you put together. By the way, thanks for porting to Godot 4! I'm always really surprised, and impressed to see how much work people put into this stuff.

Edit: 'Project Settings/Fmod' don't show up in the Project Settings menu. And I can't get sound to play from a bank file. I tried one I made, plus an example music file from the Fmod studio project. Are there any easy things i can check to troubleshoot this? None of the functions fail, but the create_event_instance returns Object#Null when I print it, so seems like something isn't set up right. It does print a statement that the initialize function was successful.

**Comment by jordigulley** [comment]
`Project Settings/Fmod` is only visible when 'Advanced Settings' is enabled in the settings viewer. The Fmod section will be at the bottom!
![image](https://user-images.githubusercontent.com/12436824/204181361-6d984169-02c6-43f0-9e88-467812548ce4.png)

Thanks for your suggestion. I see I should revisit this project and improve it for other's use. This initially started as an internal project for my own game! I'll add instructions for the dll's, improve documentation, and add some more QoL changes (such as adding Event pickers to the Inspector). I'll look into what may be causing your issue shortly!

**Comment by jordigulley** [comment]
e123447 improves error handling for this project. Can you recompile your project with the new changes and see if it tells you any better info? Hope this helps!

**Comment by jordigulley** [comment]
21e314f9d69edd0cf5a05f94a7c006abece779be closes this issue.

**Comment by StoreBoughtRocketGames** [comment]
Finally got a chance to test the recompile, and it throws a few errors. 

![image](https://user-images.githubusercontent.com/110170625/204424979-311d9f45-1edc-4bd4-9c14-20baf418d9b4.png)

Admittedly, I don't know anything about FMOD. I'm doing this because the sound guy for my project wanted to use FMOD, and I am working with Godot 4 (You're project is the only one that was geared toward 4). So if I did something dumb here, and it's obvious, apologies, that's why. 


Here's a simple piece of code I put together to test the API.
```
extends FmodManager

var music_vca:FmodVCA;
var fmod_event

# Called when the node enters the scene tree for the first time.
func _ready():
	initialize(1024, FmodManager.NORMAL);
	load_bank("Sound/FMOD/Master.bank", FmodManager.NORMAL_LOAD);

	fmod_event = Fmod.create_event_instance("Sound/FMOD/Music.bank", true, true)
	print(fmod_event)
	load_bank("Master.strings.bank", FmodManager.NORMAL_LOAD);
	Fmod.play("Sound/FMOD/Music.bank")#, FmodManager.NORMAL_LOAD);
	print('This was entered')
```

**Comment by jordigulley** [comment]
Can you take a screenshot of your project directory? The error you're getting is because Fmod could not find the bank you wanted to load.

I'm also noticing that you're creating an Event Instance using a path to a sound bank instead of a path to an Fmod Event. A quick and easy way to get the path to an event you've created is by right clicking on it and clicking `Copy Path`
![image](https://user-images.githubusercontent.com/12436824/204428749-9f709936-0b62-43b4-a575-1eb733a77fec.png)

**Comment by StoreBoughtRocketGames** [comment]
Kind of sounds like I'm just not using FMOD correctly, and it's got nothing to do with your code. I assumed a bank file was the sound file that I created. I was attempting to  test background music. I don't know if that makes any difference. The music file is a sample from FMOD. I created the master file.
![image](https://user-images.githubusercontent.com/110170625/204439476-e383c175-ed06-431a-a065-4d9ed029d78d.png)
